contracts/RollupPoB.sol

```diff
@@ -0,0 +1,331 @@
+pragma solidity ^0.6.1;
+import './RollupInterface.sol';
+import './lib/RollupPoBHelpers.sol';
+
+contract RollupPoB is RollupPoBHelpers{
+
+    // Rollup smart contract address
+    RollupInterface rollupInterface;
+
+    // First block where the first era begins
+    uint public genesisBlock;
+
+    // Maximum rollup transactions: either off-chain or on-chain transactions
+    uint public MAX_TX;
+
+    // Input snark definition
+    uint256 constant offChainHashInput = 3;
+
+    // Defines slot/era block duration
+    uint constant public DELAY_GENESIS = 1000;
+    uint32 constant public BLOCKS_PER_SLOT = 100;
+    uint32 constant public SLOTS_PER_ERA = 20;
+    uint constant public SLOT_DEADLINE = 20;
+
+    // Burn Address
+    address payable burn = address(0x00);
```

Comment:
Burn address should be a configurable parameter when the contract is deployed.
Therefore, it would be a parameter in the constructor.